app/controllers/customers_controller.rb

```diff
@@ -142,6 +142,18 @@ def forgot_password
     end
   end
 
+  def reset_token
+    token = params[:token]
+    @customer = Customer.where('token LIKE ?', token).first
+    if @customer.nil? || (Time.now).utc >= (@customer.token_created_at + 10.minutes).utc
+      flash[:notice] = "The reset password link has expired"
+      return redirect_to login_path
+    else
+      @customer.token_created_at = (Time.now).utc - 10.minutes
+    end
+    puts("The token expires at: #{(@customer.token_created_at + 10.minutes).utc}. It is currently: #{(Time.now).utc}")
```

Comment:
Why is this here?  `puts` has nowhere to output when in production.  In general there should never be `puts` anywhere in production code.  Pls remove